docs/scheduler.rst

```diff
@@ -28,6 +30,27 @@ To start a scheduler, simply run the command:
 
     airflow scheduler
 
+The scheduler is also responsible for timing out ``Task Instances`` and ``DAG Runs`` that
+have over-run their respective timeouts (if they have them). By default when scheduling new
+``DAG Runs`` the scheduler will begin at the first available interval after start date that has
+not already run, and schedule forward from there.
+
+*Added in Airflow 1.8*
```

Comment:
Please dont mention this here. Documentation will be release together with Airflow itself, so there is no need.